appTicks            : 0.5
bits                : 32.0
fs                  : 192000.0
soundSpeed          : 1500.0
boatToMic           : 1.1 
x                   : 0.0
y                   : 0.0
heading             : 0.0
sen                 : -208.0
snr_threshold       : 20
highcutoff          : 7000
lowcutoff           : 2000
use2Mic             : "true"
window_type         : "hann"
overlap             : 0.9
duration_threshold  : 0.3
plot                : "false"
plot_y_max          : 7000 
plot_y_min          : 2000
